cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -236,9 +236,15 @@ resource "aws_ecs_service" "service" {
     assign_public_ip = false
   }
 
+  # The circuit breaker will mark the deployment as failed when the service does
+  # not come up or is not marked as healthy after 10 tries. This prevents the
+  # deployment from pulling the civiform image over and over, which can cause
+  # DockerHub rate limiting. The 10 tries is the minimum value given that
+  # desired_count < 20 (desired_count is probably 1 for our use case).
+  # https://docs.aws.amazon.com/AmazonECS/latest/userguide/deployment-circuit-breaker.html
   deployment_circuit_breaker {
-    enable   = false
-    rollback = false
+    enable   = true
+    rollback = true
```

Comment:
**thought:** I'm still thinking about whether it makes sense to turn on this rollback functionality if it works with it off. With the last deployment issue, it would sometimes work and other times wouldn't (regardless of the version), since there was a race condition. So I almost think it may be safer to keep rollback functionality off to use the deployment that was successful in the case that we can't create another successful deployment. Thoughts?

Reply:
Ohh that makes sense! Thanks!